ftests/core/src/main/java/org/commonjava/indy/ftest/core/content/ProxyRemoteKojiContentTest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (C) 2011 Red Hat, Inc. (jdcasey@commonjava.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.ftest.core.content;
+
+import org.commonjava.indy.client.core.helper.PathInfo;
+import org.commonjava.indy.ftest.core.AbstractContentManagementTest;
+import org.commonjava.indy.model.core.RemoteRepository;
+import org.commonjava.test.http.expect.ExpectationServer;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.commonjava.indy.model.core.StoreType.group;
+import static org.commonjava.indy.model.core.StoreType.remote;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+public class ProxyRemoteKojiContentTest
+        extends AbstractContentManagementTest
+{
+
+    @Override
+    protected boolean createStandardTestStructures()
+    {
+        return false;
+    }
+
+    /**
+     * Accessing an existing external Koji site and retrieve artifact, e.g.,
+     * http://download.eng.bos.redhat.com/brewroot/packages/org.jboss.resteasy-tjws/3.0.17.Final/1/maven/org/jboss/resteasy/tjws/3.0.17.Final/tjws-3.0.17.Final.pom
+     *
+     * TODO: How to initiate Koji config?
```

Comment:
Eventually, we will have a CGI in the koji-dojo project (https://github.com/release-engineering/koji-dojo) that will enable a test like this to POST some sort of build bundle and have it recorded in Brew as an actual build. This will allow us to setup inintial conditions (like in a @Before method) to seed the system with the builds we will query for.

If you're interested in how we will make use of a docker test appliance like that, see the kojiji project (https://github.com/release-engineering/kojiji). It has integration tests that use the koji-dojo. However, it suffers from the same missing feature, and can't be integration tested as extensively as we'd like.

FWIW, I've done some of the research for which parts of Koji would have to be borrowed to make a CGI like this; it's under `research/` in the koji-dojo project.